IntegrationTests/Services/AWSKinesisIntegrationTests/KinesisTests.swift

```diff
@@ -22,7 +22,7 @@ class KinesisTests: XCTestCase {
         // Resources will be cleaned up before the test concludes, pass or fail, unless the test crashes.
 
         let streamName = UUID().uuidString
-        let client = try KinesisClient(region: "us-west-2")
+        let client = try await KinesisClient(region: "us-west-2")
```

Comment:
Clarifying question: is the constructor become async like this the norm with these changes?

Reply:
Yes. If user doesn't provide region or other settings to the client constructor, then I think it makes sense for it to be asynchronous because it will try to fetch the settings from local config file.